docs/reference/plugins/_python_common.rst

```diff
@@ -0,0 +1,28 @@
+
+Dependencies
```

Comment:
This part is mostly unchanged, some minor bits to talk about both plugins instead of just the `python` one